dashboard/app/models/unit_group.rb

```diff
@@ -713,14 +713,4 @@ def prevent_course_version_change?
       default_units.any? {|s| s.prevent_course_version_change?}
     # rubocop:enable Style/SymbolProc
   end
-
-  # Look through all of the objects with the specified family name which have
-  # a stable published_state, and return the one with the latest version year.
-  def self.latest_stable(family_name)
```

Comment:
`latest_stable_version` is the same method and this is not being used so removing.